docs/operators/jdbcPOJOInsertOutputOperator.md

```diff
@@ -0,0 +1,159 @@
+
+## Overview
+The main features of this operator are persisting data to the database table and fault tolerance. This operator creates a transaction at the start of window, executes batches of sql updates, and closes the transaction at the end of the window. Each tuple corresponds to an SQL update statement. The operator groups the updates in a batch and submits them with one call to the database. Batch processing improves performance considerably. The size of a batch is configured by batchSize property. The tuples in a window are stored in check-pointed collection which is cleared in the endWindow(). This is needed for the recovery. The operator writes a tuple exactly once in the database, which is why only when all the updates are executed, the transaction is committed in the end window call. 
+This operator internally extends the functionality of AbstractPassThruTransactionableStoreOutputOperator.This is the base implementation for a pass-through output adapter of a transactional store.This implementation provides the "exactly once" guarantee."Pass-through" means it does not wait for end window to write to the store. It will begin transaction at begin window and write to the store as the tuples come and commit the transaction at end window.
+
+**Note**: For enforcing exactly once semantics a table named 'dt_meta' must exist in the database. The SQL to create the same is as follows
+"CREATE TABLE IF NOT EXISTS dt_meta (dt_app_id VARCHAR(100) NOT NULL, dt_operator_id INT NOT NULL, dt_window BIGINT NOT NULL, UNIQUE(dt_app_id,dt_operator_id,dt_window))". The operator will fail if this table is missing.
+
+**Note**: Additionally this operator assumes that the underlying database/table in which records are to be added supports transactions. If the database/table does not supports transactions then a tuple may be inserted in a table more than once in case of auto recovery from a failure (violation of exactly once semantics).
+
+## Operator Information
+1. Operator location: ***malhar-library***
+2. Available since: ***0.9.4***
+3. Java Packages:
+    * Operator: ***[com.datatorrent.lib.db.jdbc.AbstractJdbcTransactionableOutputOperator](https://www.datatorrent.com/docs/apidocs/com/datatorrent/lib/db/jdbc/AbstractJdbcTransactionableOutputOperator.html)***
+
+## How to Use?
+The tuple type in the abstract class is a generic parameter. Concrete subclasses need to choose an appropriate class (such as String or an appropriate concrete java class). Also implement a couple of abstract methods (if not using AbstractJdbcPOJOOutputOperator): `setStatementParameters(PreparedStatement statement, T tuple)` to set the parameter of the insert/update statement (which is a PreparedStatement) with values from the tuple and `getUpdateCommand()` the sql statement to update a tuple in the database. Note that these methods are also defined in AbstractJdbcPOJOOutputOperator.
```

Comment:
in the abstract class -- which one ?
This whole paragraph is not clear because this document is about a concrete class so the discussion about abstract classes is confusing; if your are talking about a specific superclass, you need to make that clear and also clarify why you are discussing the base class.